packages/extensions/openapi-to-typespec/test/schemaRegistry/readme.md

```diff
@@ -1,6 +1,6 @@
 ```yaml
 input-file:
-  - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2021-10/schemaregistry.json
```

Comment:
can we change to a fixed commit to prevent further problem?

Reply:
resolve by using fixed url of original spec